Use_Cases/Related_Lists/README.md

```diff
@@ -0,0 +1,26 @@
+# Better Related Lists
+
+The Lightning Account detail page is extended by using Skuid to display related Opportunities and Cases with advanced filtering, sorting, and interactive capabilities. The related lists have multi-field sorting, 10+ columns that can be shown/hidden by individual end users, dynamic filtering of opportunities, two views of Cases (one tab displays open cases, second tab displays closed cases), and a dashboard displaying real time opportunity information.
+
+<img src="OppList.png" width="300"></img>
+
+<img src="OppDashboard.png" width="300"></img>
+
+
+## Instructions
+- Page API:  V2
+- Data source: Uses default Salesforce data source
+- Design system: [Download this Design System file](Skuid_Appetizer.designsystem).  Use the Import function on the Design System page to add this system to your org. 
+- Page XML:  [Copy the XML from this page](FilteredLists.xml), or save it as an XML file, and upload it as a new page in your Salesforce Org.  
+- You can previewing this page by itself to see the Skuid functionality,  but to see it on the Lightning Detail page you will have to add the Skuid Page Component to a Lightning Page,  and refer to this page there. 
+
+## Notes
+
+- End users can select which columns they want to display by clicking the 3 dot menu at the top right of the table. They also have the ability to export all the records in the table into a .csv. End users can inline edit the related records across both the Opportunity and Case object simultaneously - saving changes on both objects with just one click.
```

Comment:
For file types, capitalize the extension and leave out the `.`

Also pluralization:

```suggestion
- End users can select which columns they want to display by clicking the 3 dot menu at the top right of the table. They also have the ability to export all the records in the table into a CSV file. End users can inline edit the related records across both the Opportunity and Case objects simultaneously - saving changes on both objects with just one click.
```